[BLOG] Create Announcing Scientific Python Accessibility Events (#875)
Quansight/Quansight-website: apps/labs/posts/announcing-scientific-python-accessibility-events.md

---
title: 'Announcing Scientific Python Accessibility Events'
published: September 5, 2024
authors: [isabela-presedo-floyd]
description: 'I am happy to announce two upcoming public events focused on helping the scientific Python ecosystem develop their accessibility skills before the new year.'
category: [Access-centered, OSS Experience]
featuredImage:
  src: /posts/announcing-scientific-python-accessibility-events/blog_feature_1.png

>   alt: 'A tree branch extends from the screen of a laptop with a bar chart.'

Reviewer: ```suggestion
  alt: 'A tree branch extends from a laptop screen. There is a bar chart on the screen.'
```

hero:
  imageSrc: /posts/announcing-scientific-python-accessibility-events/blog_feature_1.png
  imageAlt: 'A tree branch extends from the screen of a laptop with a bar chart.'
---

I am happy to announce two upcoming public events focused on helping the scientific Python ecosystem develop their accessibility skills before the new year. Supported by the [Chan Zuckerberg Initiative’s Essential Open-source Software for Science grants](https://chanzuckerberg.com/eoss/proposals/), these events pair with the [Scientific Python project](https://scientific-python.org/) to address popularly requested accessibility recommendations and gaps the team has found working in the ecosystem.

## What’s happening at these events?

Each event has its own theme and will cover different, though overlapping, accessibility considerations. You can attend one or both events, no prerequisites required.

### Accessibility in your digital day-to-day

> For anyone who spends part of their day online, figuring out how to incorporate accessibility can leave many question marks. Advice, if you can find it all, is often vague enough or detailed enough to make action impossible. This event is designed to align practical examples of day-to-day digital activities—like making a calendar invite or sending things to your friends—with small habits that can make identifying and remedying accessibility pitfalls clear. All in the style of a choose-your-own-adventure story with audience polling!

Reviewer 2: ```suggestion
For anyone who spends part of their day online, figuring out how to incorporate accessibility can leave many question marks, including how to get started. If you can find it all, advice is often vague enough or detailed enough to make action impossible. This event is designed to align practical examples of day-to-day digital activities—like making a calendar invite or sending things to your friends—with small habits that can make identifying and remedying accessibility pitfalls clear. All in the style of a choose-your-own-adventure story with audience polling!
```

> For anyone who spends part of their day online, figuring out how to incorporate accessibility can leave many question marks, including how to get started. If you can find it all, advice is often vague enough or detailed enough to make action impossible. This event is designed to align practical examples of day-to-day digital activities—like making a calendar invite or sending things to your friends—with small habits that can make identifying and remedying accessibility pitfalls clear. All in the style of a choose-your-own-adventure story with audience polling!

Reviewer: ```suggestion
For anyone who spends part of their day online, figuring out how to incorporate accessibility can leave many question marks, including how to get started. If you can find it at all, advice is often vague enough (or so overwhelmingly detailed) to make action impossible. This event is designed to align practical examples of day-to-day digital activities—like making a calendar invite or sending things to your friends—with small habits that can make identifying and remedying accessibility pitfalls clear. All in the style of a choose-your-own-adventure story with audience polling!
```

**When:** Tuesday, 17 September, 2024 at 8 AM Pacific Time ([in your timezone](https://arewemeetingyet.com/Los%20Angeles/2024-09-17/08:00/Accessibility%20in%20your%20digital%20day-to-day%20with%20Scientific%20Python%20and%20Quansight%20Labs))

**What attendees should know:** No prior knowledge needed.

**[Register for the first event here](https://us06web.zoom.us/meeting/register/tZIvd-qprjIiH90QqVSf_GKSQzqt_x1H21Ms)**

### Data visualization and accessibility

> Data visualization is a varied and often complex topic that draws on many disciplines. Many have advised on ways to make data visualizations include more options for people who are blind and low vision and deaf and hard of hearing. This event will compile existing recommendations to provide an abbreviated guide to the what and why behind making data visualizations available to more people.

Reviewer: ```suggestion
Data visualization is a varied and complex topic that draws on many disciplines. Many have advised on ways to make data visualizations more inclusive of people who are blind or low vision and/or deaf or hard of hearing. This event will compile existing recommendations to provide an abbreviated guide to the what and why behind making data visualizations available to more people.
```
